<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceSettingsWidget</class>
 <widget class="QWidget" name="DeviceSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="refresh_btn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QLabel" name="device_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>76</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Device</string>
   </property>
  </widget>
  <widget class="QComboBox" name="devices_cb">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>321</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="type_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>115</y>
     <width>111</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Device Type</string>
   </property>
  </widget>
  <widget class="QComboBox" name="type_cb">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>321</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="ports_cb">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>321</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="baudrate_cb">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="baudrate_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>88</y>
     <width>80</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Baudrate:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="enable_rawwave_analyser">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>106</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Use Raw Wave Analyser</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
